files/tasks/rpm-test-deps.yaml

```diff
@@ -25,3 +25,8 @@
     name: pytest-cov
   when: ansible_distribution != 'Fedora'
   become: true
+
+- name: Install sandcastle from the main branch
+  pip:
+    name: git+https://github.com/packit/sandcastle
+  become: true
```

Comment:
The name (of this file) `rpm-test-deps` suggests we install RPMs here :-)
There's also [tasks/sandcastle.yaml](https://github.com/packit/packit/blob/main/files/tasks/sandcastle.yaml) which does the same - can it be reused perhaps?

I'll take a look tomorrow if you tell me what's failing/wrong/insufficient with current workflow.

Reply:
what's wrong? me being ignorant - I haven't checked if there are already tasks which do this; let me fix! thank you for the pointer